web/src/main/webapp/js/main-template.js

```diff
@@ -158,6 +159,8 @@ $(document).ready(function (e) {
 
                 // execute query
                 initFromParams(urlParams, true);
+
+                checkInput();
```

Comment:
No, unrelated. This has to do with the order of things happening.

When the ready() function is executed, the autocomplete menu for the first 2 locations fields (empty From and To) is already finished, and still has "en" locale. One must wait until the API call to retrieve translations to know the locale.